tracker/model/user.py

```diff
@@ -22,10 +23,13 @@ class User(db.Model):
     token = db.Column(db.String(TOKEN_LENGTH), index=True, unique=True, nullable=True)
     role = db.Column(UserRole.as_type(), nullable=False, default=UserRole.reporter)
     active = db.Column(db.Boolean(), nullable=False, default=True)
+    otp_token = db.Column(db.String(OTP_TOKEN_LENGTH), index=True, nullable=True)
 
     is_authenticated = False
     is_anonymous = False
 
+    first_time_login_otp = False
```

Comment:
Should be true, since users don't have an token the first time they are created.

Reply:
I don't think we need this property